divi/src/miner.cpp

```diff
@@ -242,6 +242,7 @@ void CalculatePriority( map<uint256, vector<COrphan*> >& mapDependers, list<COrp
         } else
             vecPriority.push_back(TxPriority(dPriority, feeRate, &mi->second.GetTx()));
     }
+    return vecPriority;
 }
```

Comment:
This function was extracted but not replaced immediately. This leaves the expectation that a later commit will actually proceed to use the extracted function. It's less transparent.